src/components/Form/ExitFormDialog.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { render } from "@testing-library/react";
+import React from "react";
+
+import ExitFormDialog from "./ExitFormDialog";
+
+jest.mock("react-intl", () => ({
+  useIntl: jest.fn(() => ({
+    formatMessage: jest.fn(x => x.defaultMessage),
+  })),
+  defineMessages: jest.fn(x => x),
+}));
+
+jest.mock("@saleor/macaw-ui", () => ({
+  useStyles: jest.fn(() => () => ({})),
+  makeStyles: jest.fn(() => () => ({})),
+  DialogHeader: jest.fn(() => () => <></>),
+}));
+
+describe("ExitFormDialog", () => {
+  it("closes when ignore changes is clicked", () => {
+    // Arrange
+    const props = {
+      onClose: jest.fn(),
+      onLeave: jest.fn(),
+      isOpen: true,
+    };
+
+    // Act
+    const { getByTestId } = render(<ExitFormDialog {...props} />);
+    const button = getByTestId("ignore-changes");
```

Comment:
**suggestion**: according to [RTL](https://testing-library.com/docs/user-event/intro) documentation it seems like a way of testing user interactions

```suggestion
		import userEvent from "@testing-library/user-event";
		// the rest of the test
		const user = userEvent.setup()
    await user.click(getByTestId("ignore-changes"));
```